devicetypes/smartthings/orvibo-smoke-sensor.src/orvibo-smoke-sensor.groovy

```diff
@@ -0,0 +1,161 @@
+ /*
+  *  Copyright 2018 SmartThings
+  *
+  *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  *  use this file except in compliance with the License. You may obtain a copy
+  *  of the License at:
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  *  Unless required by applicable law or agreed to in writing, software
+  *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  *  License for the specific language governing permissions and limitations
+  *  under the License.
+  *  Author : Fen Mei / f.mei@samsung.com
+  *  Date : 2018-07-06
+  */
+
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition (name: "Orvibo Smoke Sensor", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.sensor.smoke", vid: "generic-smoke") {
+		capability "Smoke Detector"
+		capability "Sensor"
+		capability "Battery"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Alarm"
+		capability "Health Check"
+
+		fingerprint profileId: "0104",deviceId: "0402",inClusters: "0000,0001,0003,0500", outClusters: "", manufacturer: "Heiman", model: "b5db59bfd81e4f1f95dc57fdbba17931"
```

Comment:
Since we don't have a generic Zigbee Smoke Alarm yet I suggest we make this DTH the generic one. Then set the `deviceJoinName` on the fingerprint to `Orvibo Smoke Sensor` so it gets the correct name on join.